src/rpc/output_script.cpp

```diff
@@ -237,12 +237,30 @@ static RPCHelpMan deriveaddresses()
         {
             {"descriptor", RPCArg::Type::STR, RPCArg::Optional::NO, "The descriptor."},
             {"range", RPCArg::Type::RANGE, RPCArg::Optional::OMITTED_NAMED_ARG, "If a ranged descriptor is used, this specifies the end or the range (in [begin,end] notation) to derive."},
+            {"details", RPCArg::Type::BOOL, RPCArg::Default{false}, "If true, the keys of each address will be shown. Private keys will be shown only if the descriptor contains xpriv."},
```

Comment:
Booleans should typically not be positional arguments, but instead members of an "options" Object.

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/25366/commits/544c2cce910035310771f3e4979f0b59f1a62d95. Thanks for review.